<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot plot examples</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="toolBox2">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="toolBoxPage1_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>473</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Sample Settings</string>
       </attribute>
       <widget class="QPushButton" name="sampButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>239</y>
          <width>107</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reconButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>205</y>
          <width>107</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Reconnect</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="baudValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Baud Rate:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="velValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Velocity Rate:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="bValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.020000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Bias:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clrButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>274</y>
          <width>107</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="toolBoxPage2_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>352</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Cyclic Voltametry</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="toolBoxPage3_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>352</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Amperometric</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="toolBoxPage4_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>250</width>
         <height>352</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Algorithm/Concluding Conc.</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
